package.json

```diff
@@ -45,7 +45,8 @@
     "package:eject": "cd $INIT_CWD && react-scripts eject"
   },
   "resolutions": {
-    "@typescript-eslint/eslint-plugin": "^4.18.0"
+    "@typescript-eslint/eslint-plugin": "^4.18.0",
+    "react-native-qrcode-scanner/react-native-permissions": "^3.6.1"
```

Comment:
This one took a while to work out. So the version of `react-native-permissions` in `react-native-qrcode-scanner` is no longer supported https://github.com/moaazsidat/react-native-qrcode-scanner/issues/403

This seems to fix things simply enough, though it would be nice if we could add comments to a `package.json` file explaining why

Reply:
Hmm, that repo is archived. Should we jump to another one? Looks like plain old react native camera supports QR code reading: https://react-native-camera.github.io/react-native-camera/docs/rncamera#onbarcoderead and https://react-native-camera.github.io/react-native-camera/docs/rncamera#ongooglevisionbarcodesdetected

Reply:
...which is also archived. The replacement repo is https://github.com/mrousavy/react-native-vision-camera

Reply:
Discussed, it's completely understandable that a package we want to add is archived, the rest of our packages are ~7 years old. Happy to leave swapping this out for later.